cmd/k8s-operator/operator.go

```diff
@@ -222,11 +229,44 @@ waitOnline:
 	}
 
 	startlog.Infof("Startup complete, operator running")
+	if shouldRunAuthProxy == "true" {
+		authProxyListener, err := s.Listen("tcp", ":443")
+		if err != nil {
+			startlog.Fatalf("could not listen on :443: %v", err)
+		}
+		go runAuthProxy(lc, authProxyListener)
+	}
 	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
 		startlog.Fatalf("could not start manager: %v", err)
 	}
 }
 
+func runAuthProxy(lc *tailscale.LocalClient, ls net.Listener) {
+	u, err := url.Parse(fmt.Sprintf("https://%s:%s", os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT_HTTPS")))
+	if err != nil {
+		panic(err)
```

Comment:
log.fatal instead of panics?

Same below

Reply:
Done